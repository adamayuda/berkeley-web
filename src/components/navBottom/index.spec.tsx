import { NavBottom } from ".";
import { render } from "@testing-library/react";

describe("src/components/navBottom/index.tsx", () => {
  test("Render NavBottom component", () => {
    const renderedNavBottom = render(<NavBottom />);
    expect(renderedNavBottom.container).toMatchSnapshot();
  });
});
