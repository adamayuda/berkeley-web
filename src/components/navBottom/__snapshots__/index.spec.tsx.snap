// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/navBottom/index.tsx Render NavBottom component 1`] = `
<div>
  <div
    class="navBottom"
  >
    <div
      class="navGrid"
    >
      <img
        alt="LinkedTeam"
        src="test-file-stub"
      />
      <img
        alt="LinkedTeam"
        src="test-file-stub"
      />
      <img
        alt="LinkedTeam"
        src="test-file-stub"
      />
      <img
        alt="LinkedTeam"
        src="test-file-stub"
      />
      <img
        alt="LinkedTeam"
        src="test-file-stub"
      />
    </div>
  </div>
</div>
`;
